<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPLCConfig</class>
 <widget class="QDialog" name="DialogPLCConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки контроллера</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>входы</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp1">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>8 :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp8">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp2">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>9:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp9">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>3:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp3">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>10:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp10">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>4:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp4">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>11:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp11">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>5:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp5">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>12:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp12">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>6:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp6">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>13:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp13">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>7:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp7">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>14:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInp14">
          <item>
           <property name="text">
            <string>напряжение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дискр/ток</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>RS485</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>сетевой адрес:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxNetAddr">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>текущий IP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCurIP"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRead">
       <property name="text">
        <string>прочитать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWrite">
       <property name="text">
        <string>записать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Ethernet</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditIP"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>маска:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditMask"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>шлюз:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditGate"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEditInfo">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
